### ISSUE TITLE
Is it time to update the latest image to the latest Ansible version?

### ISSUE THREAD

**Issue by waweber** [troykinsella/concourse-ansible-playbook-resource#18]
Ansible is currently version 2.10, and there are a few forwards-incompatible changes that were made since 2.9. Namely, galaxy collections have a fully qualified module name that is preferred over a name in the global namespace. For example,  the `docker_container` module is now named `community.docker.docker_container`. Unfortunately, this is not supported in the version shipped in this image, so my playbooks for provisioning the machine, and the playbooks for deploying, have to be written in different syntaxes.

**Comment by troykinsella** [comment]
Absolutely it's time for an `ansible` upgrade. I'll try and find some time this week.

**Comment by troykinsella** [comment]
Resolved in #17 via simply rebuilding the Docker image.
```
↪ sudo docker run -it --rm troykinsella/concourse-ansible-playbook-resource:latest bash
bash-5.1# ansible --version
ansible 2.10.7
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.8/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.8.8 (default, Mar 15 2021, 13:10:14) [GCC 10.2.1 20201203]
  ```
